bonfire/processor.py

```diff
@@ -606,7 +606,19 @@ def _process_component(self, component_name, app_name, in_recursion):
         else:
             log.info("processing component %s", component_name)
             items = self._get_component_items(component_name)
-
+            for components in items:
```

Comment:
Actually I think the root cause is due to deploying 2 ClowdApps with the same name. The updated image tag was being set in ClowdApp A and the older image tag was being set in ClowdApp B. Since 'B' was listed in the config after 'A' ... the image tag setting got squashed once the config was applied into the namespace.